aws_cloudtrail_rules/aws_console_login_without_mfa.yml

```diff
@@ -26,6 +26,9 @@ Tests:
   -
     Name: No MFA Login
     ExpectedResult: true
+    Mocks:
+      - objectName: check_new_user
+        returnValue: False
```

Comment:
So this is interesting.  I thought that mocks only return string types, so this would be returning the string `"False"` - which is a non-empty string, and therefore should return on line 11 in your rule.  I wonder if this is a difference in the backend vs using `pat` - if you put this panther, do all the tests still pass?

Reply:
The JSON Equivalent Mocking work has not been merged yet so mocking currently only accepts strings

Using the bulk uploader with this yaml should raise a error when executing `jsoniter.Unmarshal`

Reply:
The JSON Equivalent Mocking work has not been merged yet so mocking currently only accepts strings

Using the bulk uploader with this yaml should raise a error when executing jsoniter.Unmarshal

Reply:
You are correct, I put this in Panther and the test fails.  Since that helper returns a `BOOL` is there another method we can use to test, or does that have to wait until the PR @wey-chiang  mentioned in his comment?

Reply:
I modified the helper to return a string instead of a boolean so it can be mocked. I believe returning a boolean is cleaner, so once PAT supports it I will change it back.

Reply:
That works! the other option is to check the return type from the function, and in case of a string, convert it to a bool.  Once the other PR merges then you could revert / remove the checking.  [Here's](https://github.com/panther-labs/panther-analysis/blob/33009c8ae56f4896e99de11a9bf5ff7531581cfb/standard_rules/unusual_login.py#L40) an example that does that approach.

Reply:
Thats a good call, I hadn't considered doing it that way. And based on another comment @glerb agrees. Will change accordingly